### PR TITLE
I rethemed the website a bit

### PR DESCRIPTION
**Pull request by TheRedXD**
I've decided to retheme the website (it is now dark mode in true dev fashion)

At the moment it is using assets from a.thered.sh (my assets website), you could replace them though with your own if you don't trust my website lol

### CODE REVIEW

**Comment by andrewstech** [comment]
Hi sorry for the extremely late response. I shall speak to my team about it and will keep you posted

**Comment by phenax** [comment]
This looks good. @is-a-dev/maintainers, what do you think?

**Comment by wdhdev** [comment]
I like it, but the only thing is the website uses external assets/files.

**Comment by phenax** [comment]
Good point @WilliamDavidHarrison, it'd be better if the css and the font files were served from here instead.

File: index.html [hunk 1]
```diff
@@ -1,114 +1,223 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+
+<head>
+    <meta charSet="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <title>is-a.dev | Free sub-domains for developers</title>
+    <link rel="stylesheet" href="https://a.thered.sh/assets/fonts/Roboto/font.css">
+    <link rel="stylesheet" href="https://a.thered.sh/assets/fonts/UbuntuMono/font.css">
+    <link rel="stylesheet" href="https://a.thered.sh/assets/fa/css/all.min.css">
```
Comment (phenax) [inline index.html]: It'll be better if all font and css files were served from this repo instead.

**Review by phenax — CHANGES_REQUESTED** [review]

**Comment by wdhdev** [comment]
@phenax Can the maintainers team get push access so we can merge it once it is finished if you're not online?

**Comment by TheRedXD** [comment]
I'll be fixing this up, the fa/css/all.min.css is a css file linking to fontawesome, you need to include fontawesome in your thing to get it working, the fonts are pretty easy to import, I can set everything up, just give me a bit...

File: index.html (hunk 1 above)
Comment (TheRedXD) [reply index.html]: It should be served from / instead for now, my repo has both fa and fonts there

**Comment by TheRedXD** [comment]
i fixed it, there's also a deployed demo https://theredxd.github.io/is-a-dev.github.io/

File: index.html (hunk 1 above)
Comment (TheRedXD) [reply index.html]: Update: no longer served from /, rather I use relative paths which actually work this time after too many commits lol

**Comment by wdhdev** [comment]
Is Font Awesome hosted on CDNJS? That would be better than hosting it ourselves due to the amount of lines.

**Comment by TheRedXD** [comment]
Looks like it!
https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css
I can update my instance of the site with this version

**Comment by TheRedXD** [comment]
Updated. https://theredxd.github.io/is-a-dev.github.io/

File: fonts/Mangabey/LICENSE.txt
Comment (wdhdev) [inline fonts/Mangabey/LICENSE.txt:1]: Please remove this file as it is not needed

File: fonts/Roboto/LICENSE.txt
Comment (wdhdev) [inline fonts/Roboto/LICENSE.txt:1]: Please remove this file as it is not needed

File: fonts/UbuntuMono/LICENSE.txt
Comment (wdhdev) [inline fonts/UbuntuMono/LICENSE.txt:1]: Please remove this file as it is not needed

File: index.html [hunk 2]
```diff
@@ -1,114 +1,223 @@
...
+                    src="https://cdn.buymeacoffee.com/buttons/default-orange.png" alt="Buy Me A Coffee" height="28"
+                    width="119"></a>
+            <a href="https://liberapay.com/phenax" target="_blank"><img
+                    src="https://img.shields.io/badge/liberapay-donate-yellow.svg?style=for-the-badge"
+                    alt="Liberapay recurring donation button" /></a>
+        </p>
     </main>
-  </body>
+</body>
+
```
Comment (wdhdev) [inline index.html:222]:
```suggestion
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (TheRedXD) [reply index.html:222]: It is licensing, generally you'd want to leave that in but ig I can remove it?

Comment (wdhdev) [reply index.html:222]: @TheRedXD Half of the licenses are just blank and also they aren't referenced anywhere either. We can always add attribution in the footer.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Some of these fonts are available on Google Fonts, would we be able to use those instead of hosting it ourselves?

**Comment by TheRedXD** [comment]
> Some of these fonts are available on Google Fonts, would we be able to use those instead of hosting it ourselves?

Yes, I was asked to have it serve from the repo though

**Review by wdhdev — APPROVED** [review]
👌 LGTM!
@phenax

**Comment by phenax** [comment]
@WilliamDavidHarrison Sorry for the delay, I've given write access to the maintainers team.

**Comment by wdhdev** [comment]
@phenax All good. By the way, is there a faster way to contact you if needed like Telegram or something? Also, can you give them the top maintainer role on Discord?

**Comment by phenax** [comment]
@WilliamDavidHarrison, email would be the most convinient. If it is something urgent, please mention `[URGENT]` in the subject and I'll try to get it to it sooner.

**Pull request merged**